great-idea/css/index.css

```diff
@@ -63,4 +60,173 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+nav {
+    display: flex;
+    padding: 30px 0px;
+    justify-content: space-between;
+}
+
+.nav-a-logo {
+    margin: 0;
+}
+
+nav a{
+    text-decoration: none;
+    color: darkgrey;
+    margin-right: 49px;
+    align-self: center;
+}
+
+header {
+    padding: 50px 50px;
+    display: flex;
+}
+
+h2 {
+    font-size: 25px;
+}
+
+.learn-more-span {
+    width: 130px;
+    height: 35px;
+    display: flex;
+    justify-content: center;
+    padding-top: 10px;
+    background-color: white;
+    border: 1px solid black;
+    border-radius: 5px;
+    font-size: 12px;
+    margin-top: 15px;
+}
+
+.priority-high {
+    order: -1;
+    width: 100% !important;
```

Comment:
Try to avoid using '!important'. It absolutely locks down the feature, preventing any change in the future --- unless the next developer can locate and remove it. Use something more malleable, like max-width: 100%; there are different ways to ensure a property translates (like move it above any interfering CSS).